hokusai/production.yml

```diff
@@ -30,12 +30,20 @@ spec:
       containers:
         - name: convection-web
           env:
+            - name: PORT
+              value: "8080"
+            - name: PUMA_WORKERS
+              value: "1"
+            - name: PUMA_THREAD_MIN
+              value: "5"
+            - name: PUMA_THREAD_MAX
+              value: "5"
             - name: RAILS_SERVE_STATIC_FILES
-              value: "true"
+              value: 'true'
             - name: RAILS_LOG_TO_STDOUT
-              value: "true"
+              value: 'true'
             - name: PUMA_BIND
```

Comment:
Sorry for the late review - minor but I think we prefer PORT (as introduced above) to PUMA_BIND - can change this in puma config